src/NHibernate/Impl/SessionImpl.cs

```diff
@@ -1156,48 +1156,62 @@ public object Load(System.Type entityClass, object id)
 			return Load(entityClass.FullName, id);
 		}
 
+		/// <inheritdoc />
 		public T Get<T>(object id)
 		{
-			using (BeginProcess())
-			{
-				return (T)Get(typeof(T), id);
-			}
+			return (T) Get(typeof(T), id);
 		}
 
+		/// <inheritdoc />
 		public T Get<T>(object id, LockMode lockMode)
 		{
-			using (BeginProcess())
-			{
-				return (T)Get(typeof(T), id, lockMode);
-			}
+			return (T) Get(typeof(T), id, lockMode);
 		}
 
+		/// <inheritdoc />
 		public object Get(System.Type entityClass, object id)
 		{
 			return Get(entityClass.FullName, id);
 		}
 
-		/// <summary>
-		/// Load the data for the object with the specified id into a newly created object
-		/// using "for update", if supported. A new key will be assigned to the object.
-		/// This should return an existing proxy where appropriate.
-		///
-		/// If the object does not exist in the database, null is returned.
-		/// </summary>
-		/// <param name="clazz"></param>
-		/// <param name="id"></param>
-		/// <param name="lockMode"></param>
-		/// <returns></returns>
+		/// <inheritdoc />
 		public object Get(System.Type clazz, object id, LockMode lockMode)
+		{
+			return Get(clazz.FullName, id, lockMode);
+		}
+
+		/// <inheritdoc />
```

Comment:
`<inheritdoc />` is not valid here because there is no parent.
```suggestion
		/// <inheritdoc cref="SessionExtensions.Get" />
```

Reply:
I should probably go sleeping. It's my third attempt to put this comment. 🤦

Reply:
> <inheritdoc /> is not valid here because there is no parent.

Yeah we know it's been discussed here: https://github.com/nhibernate/nhibernate-core/pull/2378#discussion_r426177054

As this class is not intended to be used by end user I think it's OK to have it as is. It will become valid when we merge this method to interface. With you suggestion it's still invalid as multiple Get extensions are present. And we need to add to TODO to remove `cref`.  So maybe leave it be...